daemonset.yaml

```diff
@@ -60,40 +60,81 @@ data:
     #!/bin/bash
 
     OS_TYPE=`uname -a`
-    RPM_TYPE=""
-    echo "Detecting OS..."
+    VERSION_AMZN="1.1-12"
```

Comment:
Let's add a comment here to make us aware that this will need to change when a new patch version is released.

Reply:
Added/updated comments